com.poweriso.PowerISO.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "com.poweriso.PowerISO",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "PowerISO",
+    "finish-args": [
+        "--device=all",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "poweriso",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir /app/lib/",
+                "cp -r ./gui/ /app/lib/poweriso/",
+                "mkdir -p /app/bin/",
+                "install -m755 launcher.sh /app/bin/PowerISO",
+                "install -m755 cli/poweriso /app/bin/poweriso",
+                "install -Dm644 com.poweriso.PowerISO.png /app/share/icons/com.poweriso.PowerISO.png",
+                "install -Dm644 com.poweriso.PowerISO.desktop /app/share/applications/com.poweriso.PowerISO.desktop",
+                "install -Dm644 com.poweriso.PowerISO.appdata.xml /app/share/metainfo/com.poweriso.PowerISO.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.poweriso.com/poweriso-x64-1.1.tar.gz",
+                    "sha256": "d30cbf69b6b3f65241b909e7e26acba133993f440f3dbdf86f362889108aab44",
+                    "dest": "gui"
+                },
+                {
+                    "type": "archive",
+                    "url": "https://www.poweriso.com/poweriso-1.5.tar.gz",
+                    "sha256": "1893189e0c8b77da179df64a64cf8e62c4216684d1ab262867c6c96faae64ee2",
+                    "strip-components": 0,
+                    "dest": "cli"
+                },
+                {
+                    "type": "file",
+                    "path": "com.poweriso.PowerISO.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html